src/main/java/seedu/duke/commands/HelpCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.duke.commands;
+
+import seedu.duke.exceptions.ModHappyException;
+import seedu.duke.tasks.ModuleList;
+import seedu.duke.util.StringConstants;
+
+public class HelpCommand extends Command {
+
+    protected static final String EXIT_COMMAND_WORD = StringConstants.EXIT_COMMAND_WORD;
+    protected static final String ADD_COMMAND_WORD = StringConstants.ADD_COMMAND_WORD;
+    protected static final String DELETE_COMMAND_WORD = StringConstants.DELETE_COMMAND_WORD;
+    protected static final String LIST_COMMAND_WORD = StringConstants.LIST_COMMAND_WORD;
+    protected static final String MARK_COMMAND_WORD = StringConstants.MARK_COMMAND_WORD;
+    protected static final String HELP_COMMAND_WORD = StringConstants.HELP_COMMAND_WORD;
+    protected static final String HELP_NOTE = StringConstants.HELP_NOTE;
+    protected static final String EXIT_HELP = StringConstants.EXIT_HELP;
+    protected static final String ADD_HELP = StringConstants.ADD_HELP;
+    protected static final String DELETE_HELP = StringConstants.DELETE_HELP;
+    protected static final String LIST_HELP = StringConstants.LIST_HELP;
+    protected static final String MARK_HELP = StringConstants.MARK_HELP;
+    protected static final String HELP_HELP = StringConstants.HELP_HELP;
+    protected static final String HELP_EXCEPTION = StringConstants.HELP_EXCEPTION;
+
+    private final String command;
+
+    public HelpCommand(String command) {
+        this.command = command;
+    }
+
+    @Override
+    public CommandResult execute(ModuleList moduleList) throws ModHappyException {
+        System.out.println(StringConstants.LINE);
+        System.out.print(HELP_NOTE);
```

Comment:
Does anyone have suggestions on where I could print out the note on compulsory and optional parameters?

Reply:
I just share how netwox does. When enter the help, it only shows the information about compulsory parameter, and when enter the compulsory followed by -h or --help, it will show the optional parameters.

Reply:
Sorry, I don't quite understand this. Could you provide an example within this context?

Reply:
For example, the if user enter “help”, it will show the just compulsory parameters. But when enter “help add” it will show detailed help for add including the optional parameters. But this may need a parser for help command, and I am not sure if it’s too complex for V1.0, because the parameters may change frequently later. 
Maybe can give a brief at the beginning to introduce the format of orders e.g indicate that the parameter in bracket is optional? Just my suggestion, you can decide the way it displays as long as it’s clear.

Reply:
I see. I changed it so that the note only appears when `help` is called.

Reply:
Pretty decent implementation, I will merge it at 11:30.